src/index.js

```diff
@@ -133,17 +133,22 @@ var middleThreshRange = {
   }
 };
 //Axis configuration
-var uniformAxisBool = {
+let uniformAxisBool = {
   ref: 'props.section5.uniformAxisBool',
   label: 'Consistent for all axis dimension values',
   type: 'boolean',
   defaultValue: true
 };
 
+let randomId = {
+  randomId: (((1 + Math.random()) * 0x10000) | 0).toString(16).substring(1)
+};
```

Comment:
this will only run once for all bullet charts

Reply:
Oh so I have to still update it in the paint section but set the variable in the component?

Reply:
you need to detect if it is already created in the paint.
use the already create random value if it exists 
or create a new one and store it otherwise.

it may be possible to add a 'init' function, but only maybe (and I did not fix out how from a quick search)